Chapters/5_Core_identifiers.md

```diff
@@ -90,7 +90,55 @@ As an example, `swh:1:rev:309cf2674ee7a0749978cf8265ab91a60aea0f7d` points to
 
 ## 5.4 Releases
 
-*TODO*
+Some revisions are more equals than others and get selected by developers as denoting important project milestones known as “releases”. Each release points to the last commit in project history corresponding to the release and carries metadata: release name and version, release message, cryptographic signatures, etc. If they're not attached to development history (e.g. if they've been imported from bare tarballs), releases can also point directly to a root directory instead of a full revision with metadata.
```

Comment:
"Some revisions are more equals than others"
I'm not sure what we mean here. I guess "Some revisions are more relevant than others".

Reply:
Thank you, we can indeed simplify the language here, by simply removing "are more equals than others and", leaving just "Some revisions get selected by developers..."
@olasd may you implement this change and update the pull request?